docs/docusaurus/docs/contributing/contributing_setup.md

```diff
@@ -56,8 +56,6 @@ python3 -m venv gx_dev
 
 source gx_dev/bin/activate
```

Comment:
Do we want to leave the pip upgrade command here without the pin?

Reply:
I've been going back and forth on this since pip will show a warning if you are not on the latest version. But I think better to upgrade than not. I'll put the command back in (without the pin).